lib/recurly/fraud.js

```diff
@@ -0,0 +1,83 @@
+import dom from '../util/dom';
+import errors from '../errors';
+import each from 'component-each';
+import bind from 'component-bind';
+
+const debug = require('debug')('recurly:fraud');
+
+export class Fraud {
+  constructor (recurly) {
+    this.recurly = recurly;
+    this.selector = this.recurly.config.fields.number;
+    this.dataCollectorInitiated = false;
+    this.form = null;
+
+    recurly.ready(bind(this, this.dataCollector));
```

Comment:
Understandable. However, every other place in RJS is using the bind component so it seems for consistency at least we would use it here. I would think that we would either remove that component altogether now, or use it's inherent abstraction to update the details of the binding process inside of it.